.circleci/src/commands/install_pods.yml

```diff
@@ -4,11 +4,12 @@ steps:
       name: Restore pods
       keys:
         - v1-pods-{{ checksum "ios/Podfile.lock" }}
+        - v1-pods
   - run:
       name: Install pods
       command: |
         bundle exec pod update
```

Comment:
I feel like, we should remove the `pod update`.
If I understand correctly, `pod update` updates all dependencies and modifies `Podfile.lock` (I might be wrong, cannot test it :) )
Alternatively, we should abort if after `pod update`, the `Podfile.lock` changed (as compared to the git version), but I feel like this would be quite flaky, as new versions of pods may be released at any time (I suppose).
Probably, dependabot should take care of updating these dependencies "manually" (ie. in separate PRs).

Reply:
The announcement was just in january. I will create a ticket to not forget that but i think we may start in the beginning of next year. not sure if all packages will support that by default but this can be discussed in the particular issue

Reply:
Not sure if its necessary to create an issue for this now, I would guess that this will be addressed in a future flutter upgrade anyway?